packages/block-library/src/navigation/edit/index.js

```diff
@@ -225,13 +224,10 @@ function Navigation( {
 
 	const navRef = useRef();
 
-	const isDraftNavigationMenu = navigationMenu?.status === 'draft';
```

Comment:
Yes, I was exploring the idea of saving menus as drafts initially indifferent of how they're created so the user does the "final call" action when clicking save. The idea is to avoid fiddling with live site menus by mistake.

Reply:
Worth noting that there's still some code that changes the status to `publish` when saving wp_navigation entities in an editor:
https://github.com/WordPress/gutenberg/blob/54c9d811c1b0884671fb57a39df335753786a590/packages/editor/src/components/entities-saved-states/index.js#L32-L37
https://github.com/WordPress/gutenberg/blob/54c9d811c1b0884671fb57a39df335753786a590/packages/editor/src/components/entities-saved-states/index.js#L149-L157

Reply:
@draganescu Let me know what you'd like me to do. I can make a separate PR that removes all of the draft handling, which you can revert when you need it back. Or I can put `isDraftNavigationMenu` back in this PR

Reply:
I think you should proceed as things are no need for any separate PR. We'll see how trunk looks when we start changing the publish behavior 🙏🏻